### ISSUE TITLE
Tests for twisted.positioning should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7724]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7724|
|Type|enhancement|
|Created|2014-11-27 10:21:14Z|
|Branch|https://github.com/twisted/twisted/tree/positioning-testcase-names-7724-2|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7724.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/3c4/3c43076927091194ca066d92026f5e755d36170b/37c967edc4d6774f7b1381a8e1999e2b6733c559.patch) (624 bytes) - added by eeshangarg on 2014-11-27 10:25:08Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7724 7724
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_positioning_testcase_names_7724_2 branches/positioning-testcase-names-7724-2
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1417083674180043 1417083674180043
changetime__1422299198798359 1422299198798359
version__None None
owner__adiroiban adiroiban
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg removed owner|
|-|-|

Hi!

I just added a patch(bugfix_7724.patch​) that changes all of the tests in twisted.positioning to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks & regards,
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [43742]) Branching to positioning-testcase-names-7724.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

(In [43744]) Branching to positioning-testcase-names-7724-2.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @adiroiban|
|-|-|

The path was missing the news file but I have created it.
I have created a branch and sent it to buildbot.

If all is ok we can merge it.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_5"></a>|@eeshangarg commented|
|-|-|

Thanks for reviewing my patches, adiroiban! It really means a lot!

I am sure I did and I always include a .misc news file with these tickets. I tried downloading the above patch of mine and I do see the diff indicating that there is a news file, but when viewing the diff in trac, I can't see it. I remember someone telling me on IRC that trac's diff viewer does not indicate the creation of empty files.

However, if I am doing something wrong, please let me know and I'll be more careful next time! :-)

Thanks again!

Regards,
Eeshan Garg

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban commented|
|-|-|

Eeshan Garg, true. My bad.

I have no idea why ``` patch -p0 < bugfix_7724.patch​ ``` does not create those files... I guess is due to various diff formats.

I hope that test will pass and we can merge it.

Thanks again for your contribution!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_7"></a>|@adiroiban set status to `closed`|
|-|-|

(In [43766]) Merge positioning-testcase-names-7724-2: Rename test cases from twisted.positioning.

Author: eeshangarg
Reviewers: adiroiban
Fixes: [#7724](https://github.com/twisted/twisted/issues/7724)

Rename all test case names from twisted.positioning to new style names.
